src/Raven.Server/Smuggler/Documents/DatabaseSmuggler.cs

```diff
@@ -504,6 +506,31 @@ private SmugglerProgressBase.Counts ProcessRevisionDocuments(SmugglerResult resu
                 }
             }
 
+            switch (_destination)
+            {
+                case DatabaseDestination databaseDestination:
+                    using (_database.DocumentsStorage.ContextPool.AllocateOperationContext(out DocumentsOperationContext context))
+                    using (var tx = context.OpenWriteTransaction())
```

Comment:
We must _never_ do this.
All `WriteTransaction` _must_ be through the tx merger